dhis-2/dhis-web/dhis-web-importexport/src/main/webapp/dhis-web-importexport/eventImportForm.vm

```diff
@@ -47,16 +47,24 @@ $(function() {
     </td>
 </tr>
 <tr>
-	<td>$i18n.getString( "org_unit_id_scheme" )</td>
-	<td><select id="orgUnitIdScheme" name="orgUnitIdScheme" style="width:190px">
+	<td>$i18n.getString( "event_id_scheme" )</td>
+	<td><select id="eventIdScheme" name="eventIdScheme" style="width:190px">
 		<option value="UID">$i18n.getString( "uid" )</option>
-		<option value="UUID">$i18n.getString( "uuid" )</option>
 		<option value="CODE">$i18n.getString( "code" )</option>
-		<option value="NAME">$i18n.getString( "name" )</option>
     </select></td>
 </tr>
 
 <tr>
+  <td>$i18n.getString( "org_unit_id_scheme" )</td>
+  <td><select id="orgUnitIdScheme" name="orgUnitIdScheme" style="width:190px">
+    <option value="UID">$i18n.getString( "uid" )</option>
+    <option value="UUID">$i18n.getString( "uuid" )</option>
```

Comment:
UUID scheme is not available for org units unit more, no need to expose that option (if its available anywhere else in the UI, please just remove it)